common/src/main/java/org/mcaccess/minecraftaccess/MainClass.java

```diff
@@ -72,33 +68,28 @@ private static void _init() {
         MainClass.accessMenu = new AccessMenu();
         MainClass.fluidDetector = new FluidDetector();
 
+        for (KeyMapping km : KeyBindingsHandler.getInstance().getKeys()) {
+            KeyMappingRegistry.register(km);
+        }
+
+        ClientTickEvent.CLIENT_POST.register((Minecraft minecraftClient) -> {
+            clientTickEventsMethod(minecraftClient);
+        });
```

Comment:
A method reference can be used here.
```suggestion
        ClientTickEvent.CLIENT_POST.register(MainClass::clientTickEventsMethod);
```

Reply:
Did you test this code? The architectury docs were pretty specific on their implementation of the subscribers.